node/core/candidate-validation/src/lib.rs

```diff
@@ -55,13 +57,13 @@ const LOG_TARGET: &'static str = "candidate_validation";
 pub struct CandidateValidationSubsystem<S> {
 	spawn: S,
 	metrics: Metrics,
-	execution_mode: ExecutionMode,
+	isolation_strategy: IsolationStrategy,
 }
 
 impl<S> CandidateValidationSubsystem<S> {
 	/// Create a new `CandidateValidationSubsystem` with the given task spawner.
-	pub fn new(spawn: S, metrics: Metrics, execution_mode: ExecutionMode) -> Self {
-		CandidateValidationSubsystem { spawn, metrics, execution_mode }
+	pub fn new(spawn: S, metrics: Metrics, isolation_strategy: IsolationStrategy) -> Self {
```

Comment:
We use a different `IsolationStrategy` only for tests, correct? This warrants a comment.

Reply:
Oh, that's a good point!

Reply:
FWIW, we also use it for android. 

https://github.com/paritytech/polkadot/pull/1932/commits/f54adbf53ac4398e778e7f7cb900ccb089902bf4